databases/better_life/better_life.rb

```diff
@@ -60,21 +60,21 @@ def create_mindful_day(db, mindful_meditation, city,state, word)
       puts Rainbow("*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*").blink.blue
         med_number.times do
         create_mindful_day(db, mindful_meditation.sample, Faker::Address.city, Faker::Address.state, Faker::Hipster.word )
-        end
+    end
+
+    better_life = db.execute("SELECT * from better_life")
+    better_life.each do |suggestion|
```

Comment:
@Bontgoy for this assignment we were discouraged from using classes. Do you mean, I should / could have put this into a method for best practices? I just want to make sure I'm understanding correctly. Thank you!